### PR TITLE
Docs: Update Google Cloud Monitoring documentation

### LINKED ISSUE #42007: docs: update documentation for Cloud Monitoring
Update our screenshots and the names of some of the options may have also changed

### PR DESCRIPTION
**Pull request by fridgepoet**
**What this PR does / why we need it**:
Correct outdated flow, redirect to Google Cloud documentation when possible, and replace incorrect links.

**Which issue(s) this PR fixes**:
Closes #42007

**Special notes for your reviewer**:
I'd like to reduce some "step by step" instructions and screenshots that may be available already on Google Cloud documentation. Let me know what you all think.

### CODE REVIEW

**Review by sunker — COMMENTED** [review]
I think it looks great, thanks for improving this! 

I'll save the approval of this PR for the tech writers.

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 1]
```diff
@@ -14,13 +14,12 @@ Grafana ships with built-in support for Google Cloud Monitoring. Add it as a dat
 
 ## Google Cloud Monitoring settings
 
-To access Google Cloud Monitoring settings, hover your mouse over the **Configuration** (gear) icon, then click **Data Sources**, and then click the Google Cloud Monitoring data source.
+To access Google Cloud Monitoring settings, hover your mouse over the **Configuration** (gear) icon, then click **Data Sources**, click **Add data source**, then click the Google Cloud Monitoring data source.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:17]: Let's keep the "and", as it is needed here.

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 2]
```diff
@@ -44,26 +43,16 @@ Click on the links above and click the `Enable` button:
 #### Create a GCP Service Account for a Project
 
 1. Navigate to the [APIs and Services Credentials page](https://console.cloud.google.com/apis/credentials).
-1. Click on the `Create credentials` dropdown/button and choose the `Service account key` option.
+2. Click on the `Create credentials` dropdown/button and choose the `Service account` option.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:46]:
```suggestion
1. Click on the **Create credentials** dropdown/button and select the **Service account** option.
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 3]
```diff
@@ -44,26 +43,16 @@ Click on the links above and click the `Enable` button:
...
+3. Fill in a name for the service account in the Service account name field and then choose the Monitoring Viewer role from the Role dropdown:
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:47]:
```suggestion
1. In **Service account name**, enter a name for the account.
1. From the **Role** dropdown, choose the **Monitoring Viewer** role as shown in the image below:
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 4]
```diff
@@ -44,26 +43,16 @@ Click on the links above and click the `Enable` button:
...
-
-1. Click the Create button. A JSON key file will be created and downloaded to your computer. Store this file in a secure place as it allows access to your Google Cloud Monitoring data.
-1. Upload it to Grafana on the data source Configuration page. You can either upload the file or paste in the contents of the file.
-
-   {{< figure src="/static/img/docs/v71/cloudmonitoring_grafana_upload_key.png" max-width="550px" class="docs-image--no-shadow" caption="Upload service key file to Grafana" >}}
-
-1. The file contents will be encrypted and saved in the Grafana database. Don't forget to save after uploading the file!
-
-   {{< figure src="/static/img/docs/v71/cloudmonitoring_grafana_key_uploaded.png" max-width="600px" class="docs-image--no-shadow" caption="Service key file is uploaded to Grafana" >}}
+5. Continue and click Done.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:51]:
```suggestion
5. Click **Done**.
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 5]
```diff
@@ -44,26 +43,16 @@ Click on the links above and click the `Enable` button:
...
+6. Use the newly created account to [create a service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console).
+7. A JSON key file will be created and downloaded to your computer. Store this file in a secure place as it allows access to your Google Cloud Monitoring data.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:53]:
```suggestion
1. Use the newly created account to [create a service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console). A JSON key file is created and downloaded to your computer. 
1. Store this file in a secure place as it allows access to your Google Cloud Monitoring data.
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 6]
```diff
@@ -44,26 +43,16 @@ Click on the links above and click the `Enable` button:
...
+8. Upload it to Grafana on the data source Configuration page.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:54]:
```suggestion
1. Upload the key to Grafana via the data source configuration page.
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 7]
```diff
@@ -44,26 +43,16 @@ Click on the links above and click the `Enable` button:
...
+9. The file contents will be encrypted and saved in the Grafana database. Don't forget to save after uploading the file!
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:55]:
```suggestion
The file contents will be encrypted and saved in the Grafana database. Don't forget to save the file after uploading!
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 8]
```diff
@@ -217,10 +206,10 @@ The SLO query builder in the Google Cloud Monitoring data source allows you to d
 To create an SLO query, follow these steps:
 
 1. Choose the option **Service Level Objectives (SLO)** in the **Query Type** dropdown.
-1. Choose a project from the **Project** dropdown.
-1. Choose an [SLO service](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/services) from the **Service** dropdown.
-1. Choose an [SLO](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/services.serviceLevelObjectives) from the **SLO** dropdown.
-1. Choose a [time series selector](https://cloud.google.com/monitoring/service-monitoring/timeseries-selectors#ts-selector-list) from the **Selector** dropdown.
+2. Choose a project from the **Project** dropdown.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:209]: These should all be "1.". Please update to match correct format.

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 9]
```diff
@@ -327,7 +311,7 @@ Example Result: `monitoring.googleapis.com/uptime_check/http_status has this val
 
 ## Configure the data source with provisioning
 
-You can configure data sources using config files with Grafana's provisioning system. Read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../../administration/provisioning/#datasources" >}})
+You can configure data sources using config files with Grafana's provisioning system. Read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../../administration/provisioning/#data-sources" >}})
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:314]:
```suggestion
You can provision CloudWatch data source by modifying Grafana's configuration files. To learn more about provisioning and all the settings you can set, see the [provisioning documentation]({{< relref "../../administration/provisioning/#data-sources" >}})
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: docs/sources/datasources/google-cloud-monitoring/_index.md (hunk 8 above)
Comment (fridgepoet) [reply docs/sources/datasources/google-cloud-monitoring/_index.md:210]: My IDE fights me 🙃 Is there a reason they should all be 1?
Is that blocking? I can work around it if it is absolutely necessary. Let me know. 🙂

File: docs/sources/datasources/google-cloud-monitoring/_index.md (hunk 1 above)
Comment (fridgepoet) [reply docs/sources/datasources/google-cloud-monitoring/_index.md]: Done

File: docs/sources/datasources/google-cloud-monitoring/_index.md (hunk 8 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/google-cloud-monitoring/_index.md:210]: Unfortunately, yes. They all need to be "1". Thanks for checking!

Comment (fridgepoet) [reply docs/sources/datasources/google-cloud-monitoring/_index.md]: Done!

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**